config/config.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs'
+
+let configFile
+let localConfig
+
+// If project is being run locally, the developer will have their API key in the json file
+try {
+  configFile = fs.readFileSync('./config/hollowbot.config.json', 'utf8')
```

Comment:
Sorry about not being clear earlier. I was thinking `hollowbot.config.json` would be the default config file that lives outside of HollowBot's repo (and HollowBot would know where to look for it). Another way to decouple HollowBot from the config, is to pass a path to the config file as a command line argument. Something like...

```bash
ts-node src/activate.ts -c ../config.json
```